easybuild/easyblocks/p/python.py

```diff
@@ -366,9 +398,14 @@ def install_step(self):
 
         super(EB_Python, self).install_step()
 
+        # Create non-versioned symlinks for python and pip
         python_binary_path = os.path.join(self.installdir, 'bin', 'python')
         if not os.path.isfile(python_binary_path):
-            symlink(python_binary_path + self.pyshortver, python_binary_path)
+            symlink('python' + self.pyshortver, python_binary_path, use_abspath_source=False)
+        if self._has_ensure_pip():
```

Comment:
Maybe make the check a bit more direct: verify that the `pip<version>` binary you're symlinking to exists?

Reply:
Could do, but this is actually done in the sanity check already. So same as assuming `python<version>` exists, we can assume the `pip<version>` exists. During installation I'd say we can make such assumptions as they are checked later, so no problem here (symlinking to non-existing files is ok)

Reply:
Yeah, makes sense, it was just a small suggestion.